.github/workflows/ci.yml

```diff
@@ -135,6 +138,14 @@ jobs:
         cargo test --all-features --workspace
         cargo test --package cuprate-blockchain --no-default-features --features redb --features service
 
+    # Run `lychee` at:
+    # 1. The repo root (excluding `target/`)
+    # 2. Cuprate's cargo documentation
+    - name: Lychee
+      run: |
+        lychee . --exclude target
+        lychee target/doc/cuprate_*
```

Comment:
This is put inside `ci.yml` because we need access to `target/doc`.

This allows for scanning generated documentation, e.g. `rpc/types` have links that are generated with macros.